src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerReferToJudge.java

```diff
@@ -62,13 +62,9 @@ public void configure(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
                 ST_CIC_CASEWORKER, ST_CIC_SENIOR_CASEWORKER)
             .grantHistoryOnly(
                 ST_CIC_CASEWORKER,
```

Comment:
Should ST_CIC_CASEWORKER also be removed from .grantHistoryOnly as it is already in .grant?

Reply:
Thank you, you're correct.